src/Stratis.Bitcoin/AddressIndexing/AddressIndexerData.cs

```diff
@@ -0,0 +1,38 @@
+﻿using System.Collections.Generic;
+
+namespace Stratis.Bitcoin.AddressIndexing
+{
+    public class AddressIndexerData
+    {
+        /// <summary>Id required for litedb.</summary>
+        public int Id { get; set; }
+
+        public string TipHash { get; set; }
+
+        public List<AddressIndexData> AddressIndexDatas { get; set; }
+    }
+
+    public class AddressIndexData
+    {
+        public byte[] ScriptPubKeyBytes { get; set; }
+
+        public List<AddressBalanceChange> Changes { get; set; }
+    }
+
+    public class AddressBalanceChange
+    {
+        /// <summary><c>true</c> if there was a deposit to an address, <c>false</c> if it was a withdrawal.</summary>
+        public bool Deposited { get; set; }
```

Comment:
@noescape00 remember the changes I suggested here? I would rather change this an enum and have values Deposit and Withdrawal so that it is more explicit

Reply:
Bool will take less space during deserialization. Enum will consume whole integer or short

Reply:
I am also fine wth a bool, much simpler.